src/components/QuestionSubmissionForm.tsx

```diff
@@ -33,149 +49,203 @@ export function QuestionSubmissionForm({ session }: PageProps) {
         method: 'POST',
         body: JSON.stringify(data),
       });
+      if (result?.status === 200) {
+        setState(prevState => ({
+          ...prevState,
+          btnLoading: false,
+        }));
+        handleModalPopup();
+      }
     } catch (err) {
       console.log(err);
     }
-
-    console.log(result);
-
     reset();
   };
 
   return (
-    <div className='w-full rounded-md bg-white p-4 shadow-md sm:w-4/5 md:w-2/3 lg:w-1/2'>
-      <form className='space-y-8 divide-y divide-gray-200' onSubmit={handleSubmit(onSubmit)}>
-        <div className='space-y-8 divide-y divide-gray-200'>
-          <div>
+    <>
```

Comment:
Do you have a strong reason to use a fragment here?

Reply:
Actually, we need fragments to return multiple elements. Here, I don't think so we need can wrap in the main div. We can place the Transition code exactly after `</form>.`

Reply:
Nothing wrong with using a fragment, it's just it was shifting the whole html and the diff looked way more complicated than it should be. Now it looks good. Thanks :) Keeping the diff to a minimum makes the reviewer's life a little easier :smile:

Reply:
I appreciate it.